package.json

```diff
@@ -6,74 +6,52 @@
   "license": "AGPL-3.0",
   "private": true,
   "scripts": {
-    "serve": "vue-cli-service serve resources/js/src/index.ts --https --key private.key --cert private.pem --disable-host-check --hot",
-    "dev": "vue-cli-service build resources/js/src/index.ts --mode=development --modern",
-    "lint": "vue-cli-service lint resources/js/src resources/js/types",
-    "build": "vue-cli-service build resources/js/src/index.ts --mode=production",
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "lint": "eslint --ext .js --ext .vue --ext .ts resources/js",
+    "lint-fix": "eslint --ext .js --ext .vue --ext .ts resources/js --fix",
     "test": "jest --coverageDirectory=jest"
   },
   "files": [
     "resources/js/types"
   ],
   "dependencies": {
     "@tanem/svg-injector": "^10.1.19",
+    "@vue/cli-plugin-eslint": "^5.0.8",
     "@vue/composition-api": "^1.0.0-rc.5",
-    "vue": "^2.6.11",
+    "vue": "^2.6.14",
     "vue-class-component": "^7.2.3",
-    "vue-property-decorator": "^9.1.2"
+    "vue-property-decorator": "^9.1.2",
+    "vue-template-compiler": "2.6.14"
```

Comment:
Any reason why `vue-template-compiler` was moved from the `devDependencies` and changed from `^2.6.14` to `2.6.14`?

Reply:
1) moved - no reason, readded to dev
2) version changed because it is required that Vue and template-compiler would be exactly the same version

Note: when migrating to vue 2.7 I believe the template compiler will be completely removed